web/job_runs_controller_test.go

```diff
@@ -20,11 +21,42 @@ type JobRun struct {
 	ID string `json:"id"`
 }
 
+func BenchmarkJobRunsController_Index(b *testing.B) {
+	app, cleanup := cltest.NewApplication()
+	defer cleanup()
+
+	b.ResetTimer()
+	for n := 0; n < b.N; n++ {
+		j := testJobRunsControllerIndex(b, app)
```

Comment:
Seems like this set up should go before you reset the timer, outside the loop.

Reply:
Also, maybe consider renaming the function, as it is more setup than actual testing at this point.